src/views/RealTimeEvents/TabsView.vue

```diff
@@ -144,6 +146,41 @@
     await fetchFieldsWithOperator(tabPanels[tabSelectIndex.value])
   }
 
+  const sortFields = (fields) => {
```

Comment:
I'm not sure this logic should be kept in the .vue file. Maybe move it to a JS file? Where it can be properly unit tested?

Reply:
Claro, posso adicionar em um helper

Reply:
Recomendo que faça essa alteração no `src/views/RealTimeEvents/Blocks/content-filter-block.vue`